src/web/components/StickyBottomBanner/CMP.tsx

```diff
@@ -13,7 +14,12 @@ export const willShowCMP = async () => {
     }
 
     const isInUsa = (await getCountryCode()) === 'US';
+    const browserId: string | undefined = getCookie('bwid') || undefined;
```

Comment:
Not sure if this is the best method to retreive the `browserId`; could someone from DCR advise?

Reply:
I think this should be available at `window.guardian.config.ophan.browserId`?

Reply:
It's currently not set in DCR, it looks like the sign-in-gate uses the same mechanism https://github.com/guardian/dotcom-rendering/blob/35c9a86b4e6967815512a56f56b7d10085c5abe3/src/web/components/SignInGate/SignInGateSelector.tsx#L103 but looks like we do run `window.guardian.config.ophan.browserId = getCookieValue("bwid");` in `htmlTemplate.ts` so could use that?

Reply:
@jfsoul It probably is available at the location you give. However, I took it being acceptable from here;

https://github.com/guardian/dotcom-rendering/blob/master/src/web/components/SignInGate/SignInGateSelector.tsx#L103

The line before is uses `window.guardian.ophan.viewId`, but for some reason does not do the same for browserId.

Reply:
`window.guardian.config.ophan.browserId` is from the same `bwid` cookie (note, *not* viewId)

Reply:
Actually, it doesn't even seem to be defined here;

https://github.com/guardian/dotcom-rendering/blob/master/window.guardian.d.ts#L19-L28

Reply:
I suspect the `SignInGateSelector` could do with updating to use the same window property.

Reply:
@gtrufitt @jfsoul What about the types? It's not represented in either `window.guardian.ophan` or `window.guardian.config.ophan`.

https://github.com/guardian/dotcom-rendering/blob/d2f957fcb3eea6d3df7bb8f9c4fc2373e5e4dc9d/src/model/window-guardian.ts#L28

Reply:
They're all a bit of a mess at the moment and need fixing, but I'd suggest adding it to config.ophan

Reply:
We'll look at this on Dotcom as the types need correcting